lib/msal-common/src/auth/ScopeSet.ts

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ClientConfigurationError } from "../error/ClientConfigurationError";
+import { StringUtils } from "../utils/StringUtils";
+
+export class ScopeSet {
```

Comment:
question: are scopes standardized? as in, are there a common set of scopes that an app will look to gain access to, or can these basically be anything depending on the implementation?

Reply:
There are some reserved namespaces, such as openid, profile, etc., but outside of those scopes can be any string value. It is recommended to use a URI format, but this isn't a hard requirement.